packages/frontend/src/components/Home/LatestSavedCharts/index.tsx

```diff
@@ -35,23 +38,19 @@ const LatestSavedCharts: FC<{ projectUuid: string }> = ({ projectUuid }) => {
                 )
             }
         >
-            {savedCharts
-                .sort(
-                    (a, b) =>
-                        new Date(b.updatedAt).getTime() -
-                        new Date(a.updatedAt).getTime(),
-                )
-                .slice(0, 5)
```

Comment:
In the original code, we were only showing the last 5 updated charts, however with the new ActionCardList we show all of them. If we want to keep using ActionCardList, we should limit and sort the items displayed 
 
![image](https://user-images.githubusercontent.com/1983672/163346209-267414c9-d73b-4696-8ad3-e24c0858ba77.png)

Reply:
@rephus I brought `ActionCardList` here  because we want the menu options to appear in the home page, and all this logic lives within `ActionCardList`, so it is either reuse the component, or duplicate all the logic that lives inside it which would not make much sense given the whole point of components is to be able to reuse them. 
I did miss the part of limiting the number of items that will show and the sorting! Good catch! 👍  will work on it now! Thank you!!